### ISSUE TITLE
Are all borrowed properties initialized as data properties?

### ISSUE THREAD

**Issue by zenparsing** [RReverser/es-borrowed-props#2]
Based on the desugaring examples, it looks like the borrowed property is first accessed and then added to the new object as a data property.

``` js
let obj1 = {
    get foo() { return 123 }
};

let obj2 = { obj.foo };
```

In the above, `obj2` would have a "foo" data property whose value is 123, rather than a getter named "foo".  Is that right?  Is there any reason someone might expect or want the other behavior?

**Comment by RReverser** [comment]
That's a good question. I think that current behavior is less surprising as it fits existing concise property definition syntax (`{ a }`) and simply extends it for properties of objects and not just of Environment Record, but might be worth to discuss.

**Comment by zenparsing** [comment]
I agree.  It makes the most sense to think of this as an extension of concise initializers.
